src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -528,7 +528,11 @@
                 "errorWhileFetchingRepos": "Fehler beim Abrufen der Repositories. Bitte überprüfe deine Internetverbindung und öffne das Popup erneut.",
                 "404": "Template-Lösungs-Diff wurde noch nicht generiert. Bitte pusche etwas in die Template- oder Lösungs-Repositories, um ihn zu erstellen.",
                 "lineStatLabel": "Zeilen, die zwischen Vorlage und Lösung hinzugefügt/entfernt wurden",
-                "lineStatBetweenSubmissionsLabel": "Zeilen, die zwischen der aktuellen Abgabe und der vorherigen Abgabe hinzugefügt/entfernt wurden",
+                "lineStatBetweenSubmissionsLabelPart1": "Zeilen, die zwischen der aktuellen Abgabe mit Commit-Hash",
+                "lineStatBetweenSubmissionsLabelPart2": "und der vorherigen Abgabe mit Commit-Hash",
+                "lineStatBetweenSubmissionsLabelPart3": "hinzugefügt/entfernt wurden",
+                "lineStatBetweenSubmissionsLabelTemplate": "und der Vorlage hinzugefügt/entfernt wurden",
```

Comment:
Can you use placeholders instead of splitting the text into multiple translations?